bitcoin/Cargo.toml

```diff
@@ -31,8 +30,10 @@ rust_v_1_53 = []
 std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std", "bitcoin-internals/std"]
 no-std = ["core2", "bitcoin_hashes/alloc", "bitcoin_hashes/core2", "secp256k1/alloc"]
 
+# In CI we test the docs build using `--all-features`, here we should explicitly list all features
+# and all optional dependencies that we want to include in the docs build on https://docs.rs
 [package.metadata.docs.rs]
-features = [ "std", "secp-recovery", "base64", "rand", "serde", "bitcoinconsensus" ]
+features = ["std", "rand-std", "serde", "secp-recovery", "bitcoinconsensus-std", "base64", "core2"]
```

Comment:
I think docsrs has `--all-features` exposed somehow, maybe we  should use it instead since there are no more internal features such as fuzzing.

Reply:
Oh yes that would be way better. I'll look into it. Cheers